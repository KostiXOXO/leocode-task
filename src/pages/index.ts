import { default as UsersView } from './Users/Users';
export { UsersView };
