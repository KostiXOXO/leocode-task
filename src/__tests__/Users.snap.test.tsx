import React from 'react';
import { cleanup } from '@testing-library/react';
import renderer from 'react-test-renderer';
import { UsersView } from 'pages';

afterEach(cleanup);

it('Snapshot matches previous version', async () => {
	const tree = renderer.create(<UsersView />).toJSON();
	expect(tree).toMatchSnapshot();
});
