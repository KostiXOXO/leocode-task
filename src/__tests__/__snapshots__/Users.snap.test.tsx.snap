// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot matches previous version 1`] = `
<div
  className="spinner"
  data-testid="spinner"
>
  <span
    className="spinner-inner-1"
  />
  <span
    className="spinner-inner-2"
  />
  <span
    className="spinner-inner-3"
  />
</div>
`;
